### ISSUE TITLE
Laying down while standing close to a table forces you to lay down on/under it

### ISSUE THREAD

**Issue by Mnemotechnician** [Simple-Station/Einstein-Engines#1121]
# Description
Title, likely works ignoring walls and other solid obstacles. Introduced by #1057.

# Reproduction
1. Stand next to a table
2. Lay down

**Comment by Aidenkrz** [comment]
`_lookup.GetEntitiesInRange(uid, 0.3f)`
This needs to be lowered then, or add an occlusion check i suppose

**Comment by Mnemotechnician** [comment]
No, you should actually check the contacts and calculate the closest distance between two fixtures instead of doing this bandaid fix
